connection.js

```diff
@@ -1369,6 +1273,27 @@ function pluralFamilyStringToSingularString(plural) { /* {{{ */
     return pluralToConvert;
 } /* }}} */
 
+function rankedUnrankedGamesString(argNameString) { /* {{{ */
+    if (argNameString.includes("ranked") && !argNameString.includes("unranked")) {
+    // for example "boardsizesranked"
+        return "for ranked games";
+    } else if (argNameString.includes("unranked")) {
+    // for example "boardsizesunranked"
+        return "for unranked games";
+    } else {
+    // for example "boardsizes"
+        return "";
+    }
+} /* }}} */
+
+function minMaxToStrings(argNameString) { /* {{{ */
+    if (argNameString.includes("min")) {
+        return {minMax: "minimum", increaseDecrease: "increase", lowHigh: "low", isMinMax: true};
```

Comment:
`isMinMax` is not the right name. `isMinMax` implies the boolean is true when it is min and/or max. You actually mean `isMin` is true and `isMax` is false. Since we only need to check one we can omit `isMax`.

Reply:
Actually i see the code did use `minMax === "Min"` and `minMax === "Max"`, so it could make sense to return both `isMin` and `isMax`. However you can also do `!minMax.isMin`, depending on your preferences.